uint64 timestamp
string name
Position pos
uint32 rate
#TODO check possible info tags --> number of vehicles waiting